hc-nested-list-behavior.html

```diff
@@ -62,6 +62,28 @@
       'item-open-changed': 'itemOpenChanged'
     },
     /**
+     * will recursively descend down the tree closing each expanded tree node.
+     * Relies upon logic being in place to prevent two sibling nodes from being open concurrently.
+     *
+     * @param {Object} [parts = this.parts] - object containing all properties of each node
+     * @param {Array.<Object>} [items = this.parts.root] - root Object to begin closing nodes from.
+     * @return {Boolean} - returns true only if it sees a node with no children or all its child
+     *                     nodes are closed.
+     */
+    collapseTree(parts = this.parts, items = this.parts.root) {
```

Comment:
Right now children staying open when parent is toggled is expected functionality, so we can't just collapse all children down the tree when the parent is closed, which would be the most optimal, unfortunately.